internal/x509util/x509.go

```diff
@@ -0,0 +1,5 @@
+package x509util
+
+var (
+	oidExtensionSubjectAltName = []int{2, 5, 29, 17}
```

Comment:
Is this going to be used?

Reply:
Yes, it's currently used by `verify.go`.

I could put a description indeed, but it would literally be the variable name..."oid for Subject Alternative Name extension"
 
Note that the [crypto/x509](https://cs.opensource.google/go/go/+/refs/tags/go1.24.0:src/crypto/x509/x509.go;l=1110) library doesn't have a description, neither does our [c-t-go fork](https://github.com/google/certificate-transparency/blob/ebe4199bb851bcae1fe824ffd8745bb4dc70ede4/go/x509util/x509util.go#L74).

It's the only thing we need from `x509.go`.. I could put it in a different file, I've decided to leave it there to stay as close as possible to upstream.